exec/java-exec/src/main/codegen/data/Parser.tdd

```diff
@@ -39,7 +39,13 @@
     "METADATA",
     "DATABASE",
     "IF",
-    "JAR"
+    "JAR",
+    "ANALYZE",
+    "COMPUTE",
+    "ESTIMATE",
+    "STATISTICS",
+    "SAMPLE",
+    "PERCENT"
```

Comment:
@sudheeshkatkam mentioned
> Something like this came up before where a list of non reserved keyword might result in some ambiguous queries. See DRILL-2116. Also DRILL-3875.

Hence, these keywords were not added to the non-reserved keyword list. Also, I am not sure how we can preserve backward compatibility here.